sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -919,7 +919,7 @@
   public static final String ES_OR_OPERATION = "$or";
   public static final String PREV_USED_EMAIL ="prevUsedEmail";
   public static final String PREV_USED_PHONE ="prevUsedPhone";
-  public static final String MERGE = "merge";
+  public static final String MERGE_USER = "Mergeuser";
```

Comment:
Mergeuser to MergeUser

Reply:
We are using this in telemetry, there only first character is captial and all remaining is lower-case